### DISCUSSION TITLE
Can I extended refine-remix app  other than the admin tools?

### DISCUSSION THREAD

**Discussion by fzsf163** [refinedev/refine#5596] [Q&A] [answered]
is it possible to add my front end pages like home, about and so to refine when using with remix? Like keeping the admin panel separate from the actual front end pages.

**Comment by BatuhanW** [comment, marked as answer]
Hello @fzsf163, yes, definitely! We have a documentation page about using with existing projects, while we don't have remix example there yet, the idea applies to remix as well.

https://refine.dev/docs/guides-concepts/usage-with-existing-projects/#headless-examples

**Reply by fzsf163** [reply]
Thank you. I am studying that page. I am little shaky about these new staff. Thank you for being here.
